packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -120,6 +130,13 @@ function readBlobContent(content: ISummaryBlob["content"]): unknown {
 	return JSON.parse(json);
 }
 
+export const TestDataObjectType1 = "@fluid-example/test-dataStore1";
+class TestDataObject1 extends DataObject {
+	protected async initializingFromExisting(): Promise<void> {
+		throw new Error("EntryPoint not implemented");
```

Comment:
Add a comment here as to why this is throwing.

Reply:
Comment added.